content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,163 @@
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu. 
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard instances.
+
+To completely monitor a Sensu stack (Sensu backend, agent, API, and dashboard), you will need to have at least one other independent Sensu instance.
+A single Sensu stack cannot monitor itself completely because if some components are down, Sensu cannot create events.
+
+This guide requires Sensu plugins.
+For more information about using Sensu plugins, see [Install plugins with assets][16].
+
+_**NOTE**: This guide assumes you are not using Sensu clustering. The strategies in this guide are still useful for monitoring cluster members, but you should follow [Run a Sensu cluster][17] to effectively monitor clustered instances._
+
+## Monitor your Sensu backend instances
+
+The host running the `sensu-backend` should be monitored in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks/metrics and Sensu services that are not part of the Sensu event system (like the dashboard)
+* Remotely from an entirely independent Sensu stack to monitor Sensu components that must be running for Sensu to create events. 
+
+### Monitor the Sensu backend locally
+
+Monitor the host that the `sensu-backend` process runs on like any other node in your infrastructure.
+This includes checks and metrics for [CPU][1], [memory][2], [disk][3], and [networking][4].
+Find more plugins at [Bonsai, the Sensu asset index][5].
+
+### Monitor the Sensu backend remotely
+
+Monitor the `sensu-backend` server process from an independent Sensu stack.
+To do this, call Sensu's [health API endpoint][6] and use the [check-http plugin][7] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_server_port"
+  },
+  "spec": {
+    "command": "check-http.rb -h remote-api-hostname -P 4567 -p /health?consumers=1 --response-code 204",
+    "subscriptions": [
+      "monitor_remote_sensu_api"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## Monitor your Sensu agent instances
+
+**CONTENT NEEDED**
+
+## Monitor your Sensu API instances
+
+Monitor your Sensu API instances from an independent Sensu stack.
+To do this, call the port that the API is listening on using the [check-port plugin][8] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_api_port"
+  },
+  "spec": {
+    "command": "check-ports.rb -H remote-sensu-server-hostname -p 4567",
+    "subscriptions": [
+      "monitor_remote_sensu_api"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## Monitor your Sensu dashboard instances
```

Comment:
I'd ditch this section since we don't decouple the dashboard from a backend any more.

Reply:
We still want to monitor port 3000 somewhere though and most likely from another Sensu Go instance to catch firewall issues.

Reply:
Need to find out for the Dashboard portion if there are unauthenticated endpoint(s) to hit over HTTP to check for health.

Reply:
Moving to action item in document.